### PR TITLE
chore: update Admin API

### PR DESCRIPTION
**Pull request by dragarcia**

### CODE REVIEW

File: ansible/files/admin_api_scripts/activate_read_replica.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    rm -rf /var/lib/postgresql/data/*
+
+    # Download base backup
+    wal-g backup-fetch /var/lib/postgresql/data "$backup_name" --config /etc/wal-g/config.json
+
+    # Signal for DB to be a Read Replica upon restart
+    touch /var/lib/postgresql/data/standby.signal
+
+    # Ensure that downloaded backup is owned by the postgres Linux user
+    find /var/lib/postgresql/data/ -exec chown postgres:postgres {} +
```
Comment (darora) [inline ansible/files/admin_api_scripts/activate_read_replica.sh]: can just be a `chown -R postgres:postgres /var/lib/pg/data`?

File: ansible/files/admin_api_scripts/activate_read_replica.sh [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    chmod 770 /tmp/wal_fetch_dir
+    
+    backup_name=$1
+    
+    # Stop database and empty it
+    systemctl stop postgresql
+    rm -rf /var/lib/postgresql/data/*
+
+    # Download base backup
+    wal-g backup-fetch /var/lib/postgresql/data "$backup_name" --config /etc/wal-g/config.json
```
Comment (darora) [inline ansible/files/admin_api_scripts/activate_read_replica.sh:20]: is this blocking? will it cause the req to adminapi to timeout?

**Review by darora — APPROVED** [review]

File: ansible/files/admin_api_scripts/activate_read_replica.sh (hunk 2 above)
Comment (dragarcia) [reply ansible/files/admin_api_scripts/activate_read_replica.sh:20]: It won't. The function will be invoked [here](https://github.com/supabase/postgres/blob/575e85eb28adeb2e45b62ba61f0403f086f3bab4/ansible/files/admin_api_scripts/activate_read_replica.sh#L40) and run on the background.

File: ansible/files/admin_api_scripts/activate_read_replica.sh (hunk 1 above)
Comment (dragarcia) [reply ansible/files/admin_api_scripts/activate_read_replica.sh]: touché

**Pull request closed without merging**